app/controllers/application_controller.rb

```diff
@@ -18,6 +18,10 @@ def current_user
     current_publisher
   end
 
+  def current_ability
+    @current_ability ||= Ability.new(current_user, request.remote_ip)
```

Comment:
I followed these instructions to write this method https://gist.github.com/davejamesmiller/2189934